src/SprykerSdk/SyncApi/Message/SyncApiMessageFormatter.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\SyncApi\Message;
+
+/**
+ * Provides a formatter function for messages.
+ */
+class SyncApiMessageFormatter
+{
+    /**
+     * Colorize output in CLI on Linux machines.
+     *
+     * Info text will be in green, everything in double quotes will be yellow, and quotes will be removed.
+     *
+     * @param string $message
+     *
+     * @return string
+     */
+    public static function format(string $message): string
+    {
+        if (PHP_SAPI === 'cli' && strtolower(substr(PHP_OS, 0, 3)) !== 'win') {
```

Comment:
Some windows terminal emulators already support coloring
```
if (getenv('TERM') === 'xterm-color' || getenv('TERM') === 'xterm-256color') {
    // The terminal supports color
} else {
    // The terminal does not support color
}
```

This checks if the value of TERM is either xterm-color or xterm-256color, which indicates that the terminal supports 8-color or 256-color support, respectively.